l10n_it_reverse_charge/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,21 @@
+**Italiano**
+
+Modulo per gestire il reverse charge per le fatture fornitore.
```

Comment:
A proposito di "Reverse charge" potremmo pensare di usare anche nel README il corrispondente termine italiano "Inversione contabile".

Tra l'altro è usato anche nelle specifiche della fattura elettronica (N6), nella documentazione in rete vedo che vengono usati entrambi.

Un'idea potrebbe essere quella di usare, nella frase introduttiva qui sopra, la forma:

Modulo per gestire l'inversione contabile (reverse charge) nelle fatture fornitore.

In questo modo sarebbe chiaro a cosa si fa riferimento nel resto del README.

Reply:
Fatto @primes2h, in effetti 'reverse charge' è usato pochissime volte nel modulo rispetto a quello che mi aspettavo